docs/jwst/associations/level3_asn_rules.rst

```diff
@@ -61,30 +61,27 @@ follows:
 Rules
 -----
 
-All rules have as their base class :class:`DMS_Level3_Base
-<jwst.associations.lib.rules_level3_base.DMS_Level3_Base>` This class
+All rules have as their base class ``DMS_Level3_Base``. This class
 defines the association structure, enforces the DMS naming
-conventions, and defines the basic validity checks on the Level3
+conventions, and defines the basic validity checks on the Stage 3
 associations.
 
 Along with the base class, a number of mixin classes are defined.
 These mixins define some basic constraints that are found in a number
-of rules. An example is the :class:`AsnMixin_Base
-<jwst.associations.lib.rules_level3_base.AsnMixin_Base>`, which
+of rules. An example is the ``AsnMixin_Base``, which
 provides the constraints that ensure that the program identification
 and instrument are the same in each association.
 
-The rules themselves are subclasses of :class:`AsnMixin_Base
-<jwst.associations.lib.rules_level3_base.AsnMixin_Base>` and whatever
+The rules themselves are subclasses of ``AsnMixin_Base`` and whatever
 other mixin classes are necessary to build the rule. Conforming to the
 :ref:`class-naming` scheme, all the final
-Level3 association rules begin with ``Asn_``. An example is the
-:class:`Asn_Image <jwst.associations.lib.rules_level3.Asn_Image>` rule.
```

Comment:
`Asn_Image` does not exist in the module, so I make a best guess and replace it with something that actually exist. Is this correct?

Reply:
Yep, any example should do.